### PR TITLE
Explict add image field in  devcontainer.json

### PR DESCRIPTION
**Pull request by trganda**
For detail, see https://github.com/github/vscode-codeql-starter/issues/193

### CODE REVIEW

File: .devcontainer/devcontainer.json [hunk 1]
```diff
@@ -1,4 +1,5 @@
 {
+  "image": "mcr.microsoft.com/devcontainers/base:jammy",
```
Comment (aeisenberg) [inline .devcontainer/devcontainer.json:2]: Would this be better?

```suggestion
  "image": "mcr.microsoft.com/devcontainers/base:ubuntu",
```

**Review by aeisenberg — COMMENTED** [review]
Thanks for opening. Looks reasonable. Just a question.

**Comment by trganda** [comment]
> Thanks for opening. Looks reasonable. Just a question.

Tks, I have accepted it.

**Review by aeisenberg — APPROVED** [review]
Thanks. LGTM.

**Pull request merged**